bundles/binding/org.openhab.binding.ecotouch/src/main/java/org/openhab/binding/ecotouch/internal/EcoTouchBinding.java

```diff
@@ -163,10 +192,8 @@ private void handleEventType(org.openhab.core.types.State state,
 	/**
 	 * @{inheritDoc
 	 */
-	@Override
```

Comment:
I removed the newer JDK and reverted to 1.7. I cannot check this right now. But if you agree, that the function does not override a base class function, then the annotation is in fact not neccessary (or even wrong), correct?